app/navigation/stacks/jointOnboardingStackNavigator.tsx

```diff
@@ -21,5 +25,7 @@ export const JointOnboardingStackNavigator = () => (
     <Stack.Screen name="Loading" component={LoadingScreen} />
     <Stack.Screen name="WelcomeChild" component={WelcomeChildScreen} />
     <Stack.Screen name="SidekickIntro" component={SidekickIntroScreen} />
+    <Stack.Screen name="SignUpNickname" component={SignUpNicknameScreen} />
+    <Stack.Screen name="SignUpAvatar" component={SignUpAvatarScreen} />
```

Comment:
https://stackoverflow.com/questions/60115818/initial-route-params-in-react-navigation-v5
use predefined initial params